scripts/deploy.sh

```diff
@@ -33,12 +33,14 @@ function upload_assets {
   mv -f "${TLDR_LANG_ARCHIVES_DIRECTORY}"/*.zip "$SITE_HOME/assets/"
   rm -rf "$TLDR_LANG_ARCHIVES_DIRECTORY"
   cp -f "$TLDRHOME/index.json" "$SITE_HOME/assets/"
-  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages.pdf" "${SITE_HOME}/assets/tldr-book.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages*.pdf" "${SITE_HOME}/assets/"
+
 
   sha256sum \
     "${SITE_HOME}/assets/index.json" \
     "${SITE_HOME}/assets/"*.zip \
     "${SITE_HOME}/assets/tldr-book.pdf" \
+    "${SITE_HOME}/assets/tldr-book*.pdf" \
```

Comment:
Would only want this line or else `tldr-book.pdf` would get listed twice in the output:

```suggestion
    "${SITE_HOME}/assets/tldr-book*.pdf" \
```

However, right now the PDFs that are generated are all `tldr-pages*.pdf` so would need to update that as well if we don't want to keep the `tldr-book.pdf` nomenclature.